docs/Installation.md

```diff
@@ -214,6 +220,56 @@ service apache24 restart
 Use the procedure for installation on [Debian](#2-debian).
 
 
+## Serving the GUI and API from a custom base url
```

Comment:
I suggest that this section is moved after "What to do next" as a sort of appendix.

Reply:
done